manual/cypher/cypher-docs/src/docs/dev/general/compatibility.asciidoc

```diff
@@ -1,24 +1,23 @@
 [[cypher-compatibility]]
 = Compatibility
 
-Cypher is still changing rather rapidly.
-Parts of the changes are internal -- we add new pattern matchers, aggregators and optimizations or write new <<query-tuning,query planners>>, which hopefully makes your queries run faster.
+Cypher is not a static language. New versions brings new features and sometimes old features get dropped.
```

Comment:
I'm still not a fan of "brings" over "introduces" here. However, if you want to use "brings", this ought to be changed to "bring"...